src/eradiate/units.py

```diff
@@ -111,6 +111,7 @@ def spectrum(cls):
             cls.RADIANCE,
             cls.REFLECTANCE,
             cls.TRANSMITTANCE,
+            cls.ANGLE,
```

Comment:
As weird as this might seem, it is fine! The `Spectrum` type simply represents a quantity that has a spectral dependency, so there is no reason not to allow angle values.